client-datastore-rdbms/src/main/resources/schema-postgresql.sql

```diff
@@ -28,6 +28,7 @@ create table if not exists client_blocklist (
 create table if not exists client_session (
     -- postgresql     
     macAddress bigint ,
+    macAddressString varchar(100) ,
```

Comment:
What about automatic upgrade of existing tables to have the new column?
In this particular case it is not that important (because in all the deployments we store client sessions in cassandra, and the postgres table is not being actually used), but for the future - whenever the schema is modified, we need to do all we can to make the upgrade automatic and painless to the existing deployments.
In this particular case, an extra statement can be added after table creation, like 
    ALTER TABLE client_session ADD COLUMN IF NOT EXISTS macAddressString varchar(100);
And then we would need to provide values for the newly added column.

As mentioned earlier, in this particular case we can leave the script as is, but in the future please make sure the upgrades are covered.